package.json

```diff
@@ -129,7 +129,7 @@
     "lint": "cross-env EFF_NO_LINK_RULES=true eslint . --format friendly",
     "lint-docs": "prettylint {.,docs,website,website/blog}/*.md",
     "lint-dist": "eslint --no-eslintrc --no-ignore --env=browser \"dist/!(bin-prettier|index|third-party).js\"",
-    "build": "node --max-old-space-size=2048 ./scripts/build/build.js",
+    "build": "node --max-old-space-size=3072 ./scripts/build/build.js",
```

Comment:
Does the build fail without increasing this?

Reply:
yes, not enough memory

Reply:
see https://dev.azure.com/prettier/prettier/_build/results?buildId=1672

Reply:
Ouch. I wonder what’s taking so much memory…